app/src/main/res/values/strings.xml

```diff
@@ -842,6 +842,15 @@ Before uploading your changes, the app checks with a &lt;a href=\"https://www.we
     <string name="quest_camera_type_fixed">Fixed</string>
     <string name="quest_camera_type_panning">Panning</string>
 
+    <string name="quest_camp_drinking_water_title">Is drinking water available to all customers here?</string>
+    <string name="quest_camp_shower_title">Does it have showers?</string>
+    <string name="quest_camp_power_supply_title">Can any customer charge their phone/laptop here?</string>
+    <string name="quest_camp_type_title">Who may camp here?</string>
+    <string name="quest_camp_type_tents_and_caravans">Both tents and caravans</string>
+    <string name="quest_camp_type_caravans_only">Only caravans</string>
```

Comment:
You might also need to try and differentiate from static caravans/mobile homes.

Reply:
Uh, do you mean as in translation issue, and if so, do you have suggestions for better English (US) description?

https://wiki.openstreetmap.org/wiki/Key:caravans defines it as:
> "Caravan" is a British English term for specially made vehicles that have space to sleep while traveling, and often include other amenities such as electricity, basic cooking facilities and water supply. The caravan might be a trailer towed by another vehicle (ie travel trailer, camping trailer, popup camper), or a larger motorized vehicle (recreational vehicle (RV) or motorhome). 

As for more detailed mapping (e.g. [static_caravans=*](https://wiki.openstreetmap.org/wiki/Key:static_caravans), [permanent_camping=*](https://wiki.openstreetmap.org/wiki/Key:permanent_camping), [cabins=*](https://wiki.openstreetmap.org/wiki/Key:cabins) etc. I do not plan to implement them at this time; feel free to create additional PR after this one is accepted if you feel the additional tags are important enough.

Reply:
I'm concerned about someone seeing a static caravan on-site and the word caravan and picking that answer, despite it potentially not being possible to turn up with a caravan/RV. I'm the wrong side of the pond to confirm whether clarity is needed for the US people, but I'd suggest the string file wants a comment at the minimum so it isn't miss-translated by others.